lib/msf/core/post/windows/powershell.rb

```diff
@@ -340,6 +342,7 @@ def psh_exec(script, greedy_kill = true, ps_cleanup = true)
               return out
             end
             ps_output = get_ps_output(cmd_out, eof, datastore['Powershell::Post::timeout'])
+            ps_output = ps_output[/#{Regexp.escape(start)}(.*?)#{Regexp.escape(stop)}/m, 1].strip  #https://stackoverflow.com/a/9661504
```

Comment:
This is a good use of `Regexp.escape` but it shouldn't be necessary because `start` and `stop` will only ever be alphabetic characters (because you're using `Rex::Text.rand_text_alpha`) which will never need to be escaped.

Reply:
yeah that makes sense :slightly_smiling_face:, will update it.